pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: sass
-version: 1.32.11
+version: 1.32.12-dev
```

Comment:
Do we have more changes in the pipeline? If not, might as well just release this as 1.32.12.

Reply:
Makes sense, I'll just release this.